Purchases/Public/Purchases.swift

```diff
@@ -651,24 +641,26 @@ public extension Purchases {
      * This function will alias two appUserIDs together.
      *
      * - Parameter alias: The new appUserID that should be linked to the currently identified appUserID
-     * - Parameter maybeCompletionBlock: An optional completion block called when the aliasing has been successful.
+     * - Parameter maybeCompletion: An optional completion block called when the aliasing has been successful.
      * This completion block will receive an error if there's been one.
      */
-    @objc func createAlias(_ alias: String, completionBlock maybeCompletionBlock: ReceivePurchaserInfoBlock?) {
+    @available(*, deprecated, message: "use logIn instead")
+    @objc(createAlias:completionBlock:)
+    func createAlias(_ alias: String, _ maybeCompletion: ReceivePurchaserInfoBlock?) {
```

Comment:
although I love the `maybe` convention, I'm not sure we should use it for `public` methods, since it's not standard. 

Also, I think we should have the same names for objc and swift, since it'd be confusing for folks who might be migrating their obj-c classes to swift, when they find that our method naming isn't consistent

Reply:
Yeah, that's a good point. For public methods, we should probably go back to just `completion`

Reply:
Updated all `completionBlock` to `completion` and removed `maybe` for all public api, as well.